src/map/status.cpp

```diff
@@ -2218,7 +2396,9 @@ bool status_check_skilluse(struct block_list *src, struct block_list *target, ui
 			// Skill blocking.
 			if (
 				(sc->data[SC_VOLCANO] && skill_id == WZ_ICEWALL) ||
+#ifndef RENEWAL
```

Comment:
please check the current SC_ROKISWEIL
this does not block all skill enemies around the caster